README.md

```diff
@@ -45,9 +45,9 @@ Simply add the configuration below to your `rebar.config` and adjust for your pr
 
 ```erlang
 {ex_doc, [
-     {extras, [<<"README.md">>, <<"LICENSE">>]},
-     {main, <<"readme">>},
-     {source_url, <<"https://github.com/namespace/your_app">>}
+     {extras, ["README.md", "LICENSE"]},
```

Comment:
These minor changes are done per the new PR, but we do try to keep backward compatibility.